src/app/admin/hof/[id]/page.tsx

```diff
@@ -47,25 +50,68 @@ function CancelDialog({ open, handleOpen }) {
   )
 }
 
-export default function Page() {
-  const [thumbnailPreview, setThumbnailPreview] = useState(null)
+export default function Page({ params }: { params: { id: string } }) {
   const [content, setContent] = useState(null)
   const [openCancelDialog, setOpenCancelDialog] = useState(false)
-  const [openDialog, setOpenDialog] = useState(false)
+  const [noData, setNoData] = useState(false)
+  const [hof, setHof] = useState(null)
 
   const {
     register,
     handleSubmit,
-    trigger,
+    setValue,
     formState: { errors },
   } = useForm()
-  const onSubmit = async (data) => {}
+
+  const onSubmit = async (data) => {
+    const news = {
+      title: data.title,
+      thumbnail: data.thumbnail[0] || null,
+      facultyId: data.facultyId,
+    }
```

Comment:
```suggestion
    const hof= {
      title: data.title,
      thumbnail: data.thumbnail[0] || null,
      facultyId: data.facultyId,
    }
```
thiếu emailOfUser, beginningYear